bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-slider",
-  "version": "2.3.2",
+  "version": "2.4.0",
```

Comment:
What's the reason for the version bump?

Reply:
I branched off master (it had a version bump, that v5 branch doesnt have).